src/settings/Validation/engine/handlers.js

```diff
@@ -26,3 +31,20 @@ export const isNotEmptyEditor = (value = '') => {
 };
 
 export const isSelected = (value) => !isUndefined(value);
+
+export const isGreaterOrEqualThanPassedField = (fieldToCompare, value, model) => {
+  const valueToCompare = get(model, fieldToCompare);
+  return parseInt(value, 10) >= parseInt(valueToCompare, 10);
+};
+
+export const isMaximumFineValueValid = (fieldToCompare, value, model) => {
+  const valueToCompare = get(model, fieldToCompare);
+  return parseInt(value, 10) > 0 && parseInt(valueToCompare, 10) !== 0;
+};
+
+export const isFineExists = (value, model) => {
+  const overdueFine = get(model, 'overdueFine.quantity');
+  const overdueRecallFine = get(model, 'overdueRecallFine.quantity');
+
+  return parseInt(overdueFine, 10) > 0 || parseInt(overdueRecallFine, 10) > 0;
+};
```

Comment:
The name `isFineExists` is a little awkward. If you want to stick with `is`/`has` prefixes for boolean methods, how about `hasFine`?